GothicVRProject/GothicVR
# Bugfix/use item destroy

# To test
1. Start Game with Buddler(554) and Seraphia(110)
2. Check if Seraphia is brushing
3. Check if Buddler is using items to eat and drink AND if they're gone before pulling out a new item from hip
4. Bonus: Walk towards the Buddler and listen if the sounds of drinking and eating are there and at the right time. Set spawnpoint to: _FP_CAMPFIRE_D_2_

Hints:
* I reworked the Lab in this PR to test my update. Feel free to spawn Seraphia/Bloodwyn/Buddler there as well and check their animations.

Boundaries:
* Eat animation on mouth isn't implemented yet.
